apps/jetbrains.py

```diff
@@ -510,12 +512,20 @@ def build_text_action_keymap():
         "extract parameter": idea("action IntroduceParameter"),
         "extract interface": idea("action ExtractInterface"),
         "extract method": idea("action ExtractMethod"),
+        "refactor in line": idea("action Inline"),
+        "refactor move": idea("action Move"),
+        "refactor rename": idea("action RenameElement"),
+        "rename file": idea("action RenameFile"),
+        "(organize | optimize) imports": idea("action OptimizeImports"),
```

Comment:
I think this might fit better as "fix imports" to align with "fix formatting"

Reply:
thats fine, i used the name organize imports because thats what jetbrains calls it as the action name, in the menus and in the action list. im ok either way